README.md

```diff
@@ -21,12 +21,27 @@ I like emoji. I like ’em a lot. Programming, code, geeks/nerds, open source, a
 1. **IMPERATIVE** ↓
 	- Make your Git commit messages imperative.
 	- Write commit message like you're giving an order.
-	- E.g. Use ✅ `Add` instead of ❌ `Added`.
-	- E.g. Use ✅ `Create` instead of ❌ `Creating`.
+	
+	_For Example:_
```

Comment:
This again makes the rules look bigger and complex like many other repos out there. I appreciate the PR just don't think it's improving things here.

Can you answer what's wrong with using `E.g.` instead of `_For Example:_`?